templates/todo/projects/csharp-cosmos-sql/README.md

```diff
@@ -191,19 +181,7 @@ This template uses [Azure Key Vault](https://docs.microsoft.com/azure/key-vault/
 
 ## Uninstall
 
-To uninstall the Azure Developer CLI:
-
-Windows:
-
-```
-powershell -ex AllSigned -c "Invoke-RestMethod 'https://aka.ms/uninstall-azd.ps1' | Invoke-Expression"
-```
-
-Linux/MacOS:
-
-```
-curl -fsSL https://aka.ms/uninstall-azd.sh | bash
-```
+To remove the Azure Developer CLI, refer to [uninstall Azure Developer CLI](https://aka.ms/azd-install?tabs=baremetal%2Cwindows#uninstall-azd).
```

Comment:
Can we pin to a generic uninstall link or does it have to be windows?

Reply:
We don't have an uninstall page in Dev Hub and I have tried without "windows", won't jump to section if not included.